cpu/driver/runtest.sh

```diff
@@ -25,6 +25,25 @@ TMPDIR=/var/tmp/$(date +"%Y%m%d%H%M%S")
 
 source $CDIR/../../cpu/common/libutil.sh
 
+NON_PSTATE_PROCESSORS="26 31 46"
+check_pstate_support()
+{
+    typeset model=$(lscpu | grep Model: | awk '{print $2}')
+    if [ -z "$model" ]; then
+	cki_log "unable to determine cpu model"
+	return 1
```

Comment:
@darcari would you consider this a kernel bug or a setup issue, just wanted to confirm it should be a fail in this case ?

Reply:
I don't really think it is possible.